test/test_utils.jl

```diff
@@ -0,0 +1,15 @@
+"""
+Compare numerical and automatic gradient.
+`f` has to be a scalar valued function.
+
+Use `ChainRulesTestUtils.rrule_test` instead 
+if the rrule is explicitly defined.
+"""
+function autodiff_test(f, xs...; atol=1e-9, rtol=1e-9)
```

Comment:
I wasn't sure at first why this was needed. Perhaps `zygote_test_gradient` might be a bit clearer as a name?